torch/fx/experimental/symbolic_shapes.py

```diff
@@ -18,6 +18,23 @@
 
 SYM_FUNCTION_MODE = None
 
+math_lib = torch.library.Library("math", "DEF")
+
+math_lib.define("size(Tensor x, int dim) -> int")
+math_lib.define("stride(Tensor x, int dim) -> int")
+math_lib.define("mul(int a, int b) -> int")
+
+def size(x, dim):
+    return x.size(dim)
```

Comment:
Why aren't these just the aten.size ops?

Reply:
that's a ... good point lol.